src/components/chat-input/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import colors from "tailwindcss/colors";
+import { SendArrow } from "@pb/components/icons";
+
+type ChatInputProps = {
+  onSendMessage: (message: string) => void;
+};
+
+const ChatInput = ({ onSendMessage }: ChatInputProps) => {
+  const [message, setMessage] = useState("");
+
+  const handleMessageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setMessage(e.target.value);
+  };
+
+  const handleSendMessage = () => {
+    if (message.trim() !== "") {
+      onSendMessage(message);
+      setMessage("");
+    }
+  };
+
+  return (
+    <div className="bottom-0 flex items-center">
+      <input
+        type="text"
+        className="flex-grow rounded-l-md border border-base-300 bg-base-100 px-4 py-2 focus:outline-none"
+        placeholder="Type your message..."
+        value={message}
+        onChange={handleMessageChange}
+      />
```

Comment:
Would you kindly utilize the base component located in the **base-input** folder for the input fields to ensure reusability? Additionally, please ensure that the **Enter** key press event is properly handled to send the message when the **enter** button is clicked. Thank you.

Reply:
Could you please explain this part more for me ?
"Would you kindly utilize the base component located in the base-input folder for the input fields to ensure reusability?"

Reply:
Apologies for any confusion. Instead of adding a new input element, could you please use the **base-input** component?